template/private-npm-package/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "<%- fullModuleName %>",
+  "description": "<%- description %>",
+  "version": "0.0.0-semantically-released",
+  "main": "lib-commonjs",
+  "module": "lib-es2015",
+  "typings": "lib-types",
+  "files": [
+    "lib"
```

Comment:
I'd prefer the new patterns we have in e.g. [Koala](https://github.com/seek-oss/koala/blob/5fb023531f8a8fc82fc06e343b0ca2c893f13e88/package.json#L21-L25). It excludes the TypeScript incremental compilation `.tsbuildinfo` which is quite massive and unnecessary